Components/button/button.css

```diff
@@ -0,0 +1,51 @@
+.btn-default{

```

Comment:
you might want to consider giving a min width to button as I could see for small texts it turns out to be very small as it is taking the width of content inside .